vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java

```diff
@@ -0,0 +1,108 @@
+package org.semanticweb.vlog4j.examples.graal;
+
```

Comment:
This file does not contain the license header. The header is automatically generated by running a maven build.

Reply:
Added license header.